README.md

```diff
@@ -0,0 +1,9 @@
+# WASI.dev
+
+WASI.dev is an introduction to [WASI](https://github.com/WebAssembly/WASI/tree/main) for users of all backgrounds. The site is under active development by volunteers coordinated through the Bytecode Alliance's Special Interest Group for Documentation. If you're interested in getting involved, you can find us on the SIG Documentation channel of the [Bytecode Alliance's Zulip server](https://bytecodealliance.zulipchat.com/), or [contribute on the WASI.dev GitHub repo](https://github.com/bytecodealliance/wasi.dev).
+
+### Local development
+
+This website is built using [Docusaurus](https://docusaurus.io/). After forking and checking out the `dev` branch of this repository, you can build and start a development server with `npm install` and `npm start`. Most changes are reflected live without having to restart the server. Run `npm build` to produce the site content, and open a PR to the `gh-pages` branch of the `bytecodealliance/wasi.dev` repo to contribute.
```

Comment:
Is `dev` the right branch? I was expecting `main`

Reply:
Makes sense, updated to `main`